script.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+const li_element = document.createElement("li");
+li_element.textContent = "これです";
```

Comment:
jsでは、あまりスネークケースを変数名にしません。何か参考にしてそうしましたか？もしそうならそのリンクを教えてください。
```suggestion
liElement.textContent = "これです";
```

Reply:
スネークケースについては特に参考にしていません。申し訳ございません。classのように記述してしいまいました。